File: 2016/gara3/primo/att/primo.c
#include <stdio.h>
#include <assert.h>

int primi(char S[]) {
    // Mettete qui il codice della soluzione
    return 42;
}


#define MAXS 7
char S[MAXS + 1];

int main() {
    FILE *fr, *fw;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%s", S));

    fprintf(fw, "%d\n", primi(S));
    fclose(fr);
    fclose(fw);
    return 0;
}

File: 2016/gara3/primo/att/primo.pas
function primi(var S: string): longint;
begin
    (* Mettete qui il codice della soluzione *)
    primi := 42;
end;


var
    S : string;
    fr, fw : text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, S);

    writeln(fw, primi(S));
    close(fr);
    close(fw);
end.
